packages/@okta/vuepress-site/docs/guides/pwd-optional-overview/main/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Password optional overview
+---
+
+## Overview
+
+Most hacking-related breaches today involve weak or stolen passwords and credential security theft. Passwords are a security vulnerability because they can be easily stolen and are prone to phishing attacks. They're also not convenient either. Users often need to remember multiple passwords, change them regularly, and either memorize or store them securely somewhere.
+
+Ditch the password for a passwordless sign-in experience. Give your users the ability to authenticate with a fingerprint, YubiKey, email, phone, or another authenticator. These authenticators can be more secure – it's harder to steal a fingerprint than a password – and more convenient since users no longer have to memorize and manage their passwords.
```

Comment:
Ditch doesn't sound great.
```suggestion
Replace the password with a passwordless sign-in experience. Give your users the ability to authenticate with a fingerprint, a YubiKey, an email, a phone, or with another authenticator. These authenticators are all more secure&mdash;it's harder to steal a fingerprint than a password&mdash;and more convenient since users no longer have to memorize and manage their passwords.
```

Reply:
Updated. But did not an "a". We can talk about it.